packages/lib/services/interop/InteropService_Importer_OneNote.postprocessHtml.test.ts

```
@@ -2,8 +2,84 @@ import '../../testing/test-utils';
+
+	it('should still extract svgs for high-complexity HTML when skipping remaining postprocessing', async () => {
+		const testImportDirectory = Setting.value('tempDir');
+
+		const importer = new InteropService_Importer_OneNote();
+		await importer.init(testImportDirectory, {
+			domParser: new DOMParser(),
+			xmlSerializer: new XMLSerializer(),
+		});
+
+		const importerWithInternals = importer as unknown as {
+			postprocessGeneratedHtmlInFolder_: (baseFolder: string)=> Promise<void>;
```

ESLint

[error] 'baseFolder' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)